pyembed/src/extension.rs

```diff
@@ -73,31 +66,29 @@ pub(crate) fn get_module_state<'a>(
 ///
 /// This creates the Python module object.
 ///
-/// We don't use the macros in the cpython crate because they are somewhat
+/// We don't use the macros in the pyo3 crate because they are somewhat
 /// opinionated about how things should work. e.g. they call
 /// PyEval_InitThreads(), which is undesired. We want total control.
 #[allow(non_snake_case)]
-pub extern "C" fn PyInit_oxidized_importer() -> *mut oldpyffi::PyObject {
-    let py = unsafe { cpython::Python::assume_gil_acquired() };
+pub extern "C" fn PyInit_oxidized_importer() -> *mut pyffi::PyObject {
+    let py = unsafe { Python::assume_gil_acquired() };
```

Comment:
I actually removed the call to `PyEval_InitThreads` in https://github.com/PyO3/pyo3/pull/1630

... so you might find you're able to use the `#[pymodule]` proc macro fine?!

Reply:
I'm not sure I can. That's because I have to define `PyModuleDef::m_size` to hold module state. I _might_ be able to get a reference to the struct somehow at run-time to populate it accordingly. But since I already have this code written, I'll probably keep it as is.